docs/index.md

```diff
@@ -27,10 +27,9 @@ provider "commercetools" {
   client_id     = "<your client id>"
   client_secret = "<your client secret>"
   project_key   = "<your project key>"
-  project_key   = "<your project key>"
   scopes        = "<space seperated list of scopes>"
   api_url       = "<api url>"
-  token_url     = "<token url>"
+  auth_url      = "<auth url>"
```

Comment:
But it currently is `token_url`. See https://github.com/labd/terraform-provider-commercetools/blob/master/commercetools/provider.go#L53

I kinda agree with naming it auth_url, but then we need to update the code first :-)